### DISCUSSION TITLE
36

### DISCUSSION THREAD

**Discussion by jojoldu** [jojoldu/blog-comments#744] [Comments]
http://jojoldu.tistory.com/36

**Comment by jojoldu** [comment]
작성자: 낭군이

혹시 이 책은 우분투 기준인가요??

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

넵 우분투 기준입니다!

**Comment by jojoldu** [comment]
작성자: haha

이 책을 e 북으로 본다고 하면 편할거같나요?

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

만화책이라 e북 보시는 리더기 크기에 따라 다를것 같습니다
그래도 괜찮을것 같습니다^^

**Comment by jojoldu** [comment]
작성자: 나그네

저도 블로그에 꽤 많은 책들이 정리되어 있는데 전부 비공개입니다. 오키에서 창천항로님 글보고 질문합니다!
이런 책 내용 공개는 저작권 위험하지 않나요? 안전하다면 저도 블로그를 공개하려고 하는데요... 전 법적 문제가 있을까봐 전체 비공개로 두고 있습니다.
아니면 어떤 방법으로 정리해서 올려야 안전할까요? 저는 책 내용은 그대로 옮겨 적는데..

**Comment by jojoldu** [comment]
작성자: 창천향로 창천향로

안녕하세요 나그네님!
댓글 감사드립니다
저 같은 경우에 해당 책에 있는 내용을 그대로 사용하면 안되는걸로 알고있습니다.
그래서 보시는것처럼 모든 예제는 제가 직접 다른 코드로 재작성 하였습니다.
마찬가지로 책에 있는 글귀도 사용하지 않고 제가 이해한 대로 적었습니다.
말씀하신대로 책 내용을 그대로 옮겨적는것은 안되는것으로 알고 있습니다.
세세한 범위는 잘몰라서 이이상의 답변은 못드려 죄송합니다 ㅠ

**Comment by ssandk** [comment]
이 책 3권까지 나왔습니다.
2권 명령어 쉘스크립트 응용
3권 네트워크 보안 입문 
수고하세요~~
